scripts/build.sh

```diff
@@ -13,8 +13,8 @@ yaml-to-sqlite content.db example_csvs example_csvs.yml
 markdown-to-sqlite content.db uses for/*.md
 
 # Build plugin and tools directories
-sqlite-utils drop-table content.db plugin_repos
-sqlite-utils drop-table content.db tool_repos
+sqlite-utils drop-table content.db plugin_repos --ignore
+sqlite-utils drop-table content.db tool_repos --ignore
```

Comment:
Added `--ignore` because otherwise we would be dropping a table that didn't exist yet in our fresh `content.db` database.